README.md

```diff
@@ -2,9 +2,9 @@
 
 _**Python utilities to prepare data for the [Timelapse Feature Explorer](https://github.com/allen-cell-animated/timelapse-colorizer)**_
 
-[Timelapse Feature Explorer](https://github.com/allen-cell-animated/timelapse-colorizer) is a browser-based web app for viewing tracked segmented data.
+[Timelapse Feature Explorer](https://github.com/allen-cell-animated/timelapse-colorizer) is a browser-based web app for viewing tracked segmented data. This package provides utilities to convert time-series data to the Timelapse Feature Explorer's format.
```

Comment:
Looks like both are used on the landing page. I'm going to go with "Timelapse Feature Explorer" until someone tells me otherwise, so I'll go ahead and fix this.